CHANGELOG.rst

```diff
@@ -2,6 +2,347 @@
+
+- Automatically extract the ``org.opencontainers.image.source`` container label into the ``git.repository_url`` tag.
+
+- The experimental OTLP ingest endpoint now supports the same settings as the OpenTelemetry Collector OTLP receiver v0.43.1.
+
+- The OTLP ingest endpoint now supports the same settings as the OpenTelemetry Collector OTLP receiver v0.44.0.
+
+- The OTLP ingest endpoint can now be configured through environment variables.
+
+- The OTLP ingest endpoint now always maps conventional metric resource-level attributes to metric tags.
+
+- OTLP ingest: the ``k8s.pod.uid`` and ``container.id`` semantic conventions
+  are now used for enriching tags in OTLP metrics.
+
+- Add the ``DD_PROCESS_CONFIG_MAX_PER_MESSAGE`` env variable to set the ``process_config.max_per_message``.
+  Add the ``DD_PROCESS_CONFIG_MAX_CTR_PROCS_PER_MESSAGE`` env variable to set the ``process_config.max_ctr_procs_per_message``.
+
+- Add the ``DD_PROCESS_CONFIG_EXPVAR_PORT`` and ``DD_PROCESS_AGENT_EXPVAR_PORT`` env variables to set the ``process_config.expvar_port``.
+  Add the ``DD_PROCESS_CONFIG_CMD_PORT`` env variable to set the ``process_config.cmd_port``.
+
+- Add the ``DD_PROCESS_CONFIG_INTERNAL_PROFILING_ENABLED`` env variable to set the ``process_config.internal_profiling.enabled``.
+
+- Add the `DD_PROCESS_CONFIG_SCRUB_ARGS` and `DD_PROCESS_AGENT_SCRUB_ARGS` env variables to set the `process_config.scrub_args`.
+  Add the `DD_PROCESS_CONFIG_CUSTOM_SENSITIVE_WORDS` and `DD_PROCESS_AGENT_CUSTOM_SENSITIVE_WORDS` env variables to set the `process_config.custom_sensitive_words`.
+  Add the `DD_PROCESS_CONFIG_STRIP_PROC_ARGUMENTS` and `DD_PROCESS_AGENT_STRIP_PROC_ARGUMENTS` env variables to set the `process_config.strip_proc_arguments`.
+
+- Added `DD_PROCESS_CONFIG_WINDOWS_USE_PERF_COUNTERS` and `DD_PROCESS_AGENT_WINDOWS_USE_PERF_COUNTERS` environment variables
+
+- Add the ``DD_PROCESS_CONFIG_QUEUE_SIZE`` and ``DD_PROCESS_AGENT_QUEUE_SIZE`` env variables to set the ``process_config.queue_size``.
+  Add the ``DD_PROCESS_CONFIG_RT_QUEUE_SIZE`` and ``DD_PROCESS_AGENT_RT_QUEUE_SIZE`` env variables to set the ``process_config.rt_queue_size``.
+  Add the ``DD_PROCESS_CONFIG_PROCESS_QUEUE_BYTES`` and ``DD_PROCESS_AGENT_PROCESS_QUEUE_BYTES`` env variables to set the ``process_config.process_queue_bytes``.
+
+- Changes process payload chunking in the process Agent to take into account
+  the size of process details such as CLI and user name.
+  Adds the process_config.max_message_bytes setting for the target max (uncompressed) payload size.
+
+- When ``ec2_collect_tags`` is configured, the Agent retries API calls to gather EC2 tags before giving up.
+
+- Retry HTTP transaction when the HTTP status code is 404 (Not found).
+
+- The datadog-agent, datadog-iot-agent and datadog-dogstatsd deb packages
+  now have a versioned weak dependency on the datadog-signing-keys package
+  version 1.1.0, instead of an unversioned weak dependency.
+
```

Comment:
I think we can remove this from our changelog (since it should be included in 7.35.1 instead):

```suggestion
```

cc @bkabrda